airflow/providers/amazon/aws/transfers/hive_to_dynamodb.py

```diff
@@ -64,18 +65,18 @@ class HiveToDynamoDBOperator(BaseOperator):
     def __init__(  # pylint: disable=too-many-arguments
         self,
         *,
-        sql,
-        table_name,
-        table_keys,
-        pre_process=None,
-        pre_process_args=None,
-        pre_process_kwargs=None,
-        region_name=None,
-        schema='default',
-        hiveserver2_conn_id='hiveserver2_default',
-        aws_conn_id='aws_default',
+        sql: str,
+        table_name: str,
+        table_keys: list,
+        pre_process: Optional[Callable[[Any, Optional[list], Optional[list]], Iterable]] = None,
```

Comment:
```suggestion
        pre_process: Optional[Callable] = None,
```
No strong opinion but this long type is too much complicated for me

Reply:
Thank you for your suggestion.  I'll fix it!